src/Sarif.Viewer.VisualStudio.Core/SarifViewerPackage.cs

```diff
@@ -171,9 +171,10 @@ private async Task InitializeResultSourceHostAsync()
             string solutionPath = GetSolutionDirectoryPath();
             if (!string.IsNullOrWhiteSpace(solutionPath) && SarifViewerOption.Instance.IsGitHubAdvancedSecurityEnabled)
             {
-                this.resultSourceHost = new ResultSourceHost(solutionPath, this);
+                var resultSourceFactory = new ResultSourceFactory(solutionPath, this);
```

Comment:
>solutionPath [](http://example.com/codeflow?start=66&length=12)

In some cases the solution is not placed in the repo root path, its placed in a sub directory of repo root. E.g. {RepoRootPath}\src, does it work in this case?

Reply:
Yes it does. (It's the result source's responsibility.)